microservices/recordings/src/Service/Encoder.php

```diff
@@ -23,6 +23,8 @@ class Encoder
      * Recording created this seconds ago will be ignored
      */
     const RECORDING_AGE_MIN = 10;
+    protected const ONDEMAND = 'ondemand';
+    protected const DDI = 'ddi';
```

Comment:
2 cents, there are contacts with the same values in the Recording entity interface:

https://github.com/irontec/ivozprovider/blob/5ad4bb273592b11e6f23d2a80f727f433402c4c1/library/Ivoz/Provider/Domain/Model/Recording/RecordingInterface.php#L17-L21

Reply:
Great! Didn't notice that.
Thanks!